core/src/i18n/en.yaml

```diff
@@ -58,26 +83,34 @@ common:
     create: Create
     delete: Delete
     edit-yaml: Edit YAML
-    save: Save
-  created: Created
   delete-dialog:
     delete-confirm: Don't ask this again
     information: You've switched off removal confirmations for all resources. You can switch them back on in Preferences.
     message: Are you sure you want to remove {{type}} {{name}}?
     title: Remove {{name}}
+  errors:
+    no-permissions: You don't have enough permissions to view this content.
+    no-permissions-header: Not enough permissions
+    no-permissions-message: Contact your administrator to grant you access.
   headers:
     created: Created
     labels: Labels
+    loading: Loading...
     name: Name
+    resources: Resources
     status: Status
   labels:
     hostname: Hostname
     name: Name
     service: Service
+  messages:
+    doesnt-exist: doesn't exist.
```

Comment:
How does this look on the UI? :flushed: 
Do we tell users what it is that doesn't exist?  
Like `{{something}} doesn't exist.` ?

Reply:
yes
`{entryType} "{entryId}" {t('common.messages.loading-name')}`